runtime/POSIX/fd_init.c

```diff
@@ -18,6 +18,10 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/stat.h>
+
+static void __emit_error(const char *msg) {
+  klee_report_error(__FILE__, __LINE__, msg, "user.err");
```

Comment:
Doesn't this defeat the point of reporting file and line number? Because all invocations will be `fd_init.c:23`?

Reply:
It's static and should get inlined for everything >O0.

Reply:
But those __LINE__ things are preprocessor?

Reply:
Ah yes, of course. It was late... So we should fix that too.